### PR TITLE
Feat: Signup ui

### PR DESCRIPTION
**Pull request by Andy-Golden**
## Task
https://trello.com/c/nUXzcFBN/4-signup-ui-https-previewcolorlibcom-theme-konato-registerhtml

## Description
- Sign up ui
- Validate with yup

## Screenshot

https://github.com/Andy-Golden/find-place-to-live/assets/147120986/207abcb1-f6cd-4070-8e9c-fdea0e0c31e5

### CODE REVIEW

File: src/constants/strongPassword.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const strongPassword =
+  /^(?=.*[A-Z])(?=.*[!@#$&*])(?=.*[0-9])(?=.*[a-z].*[a-z].*[a-z]).{6}$/;
```
Comment (carl-huynh-goldenowl) [inline src/constants/strongPassword.ts]: - Why is the password length 6?
- Give more defails about your regex (Ex: Length? Uppercase letter?...)

**Review by carl-huynh-goldenowl — CHANGES_REQUESTED** [review]

**Comment by carl-huynh-goldenowl** [comment]
Need to update desc of this PR
Screen Shot 2023-11-15 at 00 00 29

**Comment by carl-huynh-goldenowl** [comment]
I will review responsive, setup storybook in other PR

**Comment by carl-huynh-goldenowl** [comment]
Please use other font

**Comment by Andy-Golden** [comment]
I will change font in branch i18n

**Review by carl-huynh-goldenowl — APPROVED** [review]
LGTM

**Pull request merged**